include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //VkBool32           hasPrimary;
+    //VkBool32           hasRender;
+    //int64_t            primaryMajor;
+    //int64_t            primaryMinor;
+    //int64_t            renderMajor;
+    //int64_t            renderMinor;
+
+    /* ProvokingVertexPropertiesEXT *//* provided by VK_EXT_provoking_vertex */
+    //VkBool32           provokingVertexModePerPipeline;
+    //VkBool32           transformFeedbackPreservesTriangleFanProvokingVertex;
+
+    /* Robustness2PropertiesEXT *//* provided by VK_EXT_robustness2 */
+    //VkDeviceSize       robustStorageBufferAccessSizeAlignment;
+    //VkDeviceSize       robustUniformBufferAccessSizeAlignment;
+
+    /* SamplerFilterMinmaxPropertiesEXT ---> MOVED TO Vulkan 1.2 Core  */
+
+    /* SampleLocationsPropertiesEXT *//* provided by VK_EXT_sample_locations */
+    //VkSampleCountFlags    sampleLocationSampleCounts;
+    //VkExtent2D            maxSampleLocationGridSize;
+    //float                 sampleLocationCoordinateRange[2];
+    //uint32_t              sampleLocationSubPixelBits;
+    //VkBool32              variableSampleLocations;
+
+    /* SubgroupSizeControlPropertiesEXT ---> MOVED TO Vulkan 1.3 Core  */
+
+    /* TexelBufferAlignmentPropertiesEXT ---> MOVED TO Vulkan 1.3 Core  */
+
+    /* TransformFeedbackPropertiesEXT *//* provided by VK_EXT_transform_feedback */
+    //uint32_t           maxTransformFeedbackStreams;
+    //uint32_t           maxTransformFeedbackBuffers;
+    //VkDeviceSize       maxTransformFeedbackBufferSize;
+    //uint32_t           maxTransformFeedbackStreamDataSize;
+    //uint32_t           maxTransformFeedbackBufferDataSize;
+    //uint32_t           maxTransformFeedbackBufferDataStride;
+    //VkBool32           transformFeedbackQueries;
+    //VkBool32           transformFeedbackStreamsLinesTriangles;
+    //VkBool32           transformFeedbackRasterizationStreamSelect;
+    //VkBool32           transformFeedbackDraw;
+
+    /* VertexAttributeDivisorPropertiesEXT *//* provided by VK_EXT_vertex_attribute_divisor */
+    //uint32_t           maxVertexAttribDivisor;
+
+    /* AccelerationStructurePropertiesKHR *//* provided by VK_KHR_acceleration_structure */
+    uint64_t           maxGeometryCount;
+    uint64_t           maxInstanceCount;
+    uint64_t           maxPrimitiveCount;
+    uint32_t           maxPerStageDescriptorAccelerationStructures;
+    uint32_t           maxPerStageDescriptorUpdateAfterBindAccelerationStructures;
+    uint32_t           maxDescriptorSetAccelerationStructures;
+    uint32_t           maxDescriptorSetUpdateAfterBindAccelerationStructures;
+    uint32_t           minAccelerationStructureScratchOffsetAlignment;
+ 
+    /* DepthStencilResolvePropertiesKHR ---> MOVED TO Vulkan 1.2 Core  */
+    /* DriverPropertiesKHR ---> MOVED TO Vulkan 1.2 Core  */
+    /* VK_KHR_fragment_shader_barycentric --> Coverage 0% --> no structs defined anywhere in vulkan headers */
```

Comment:
yeah its a March 2022 extension, the field only has one bool and I can't really make sense of it yet